tests/gnrc_dhcpv6_client_6lbr/tests-as-root/01-run.py

```diff
@@ -161,11 +158,11 @@ def testfunc(child):
     # and is still asking for a prefix delegation
     assert DHCP6OptIA_PD in pkt
 
-    assert DHCP6OptMUD in pkt
-    mud_option = pkt[DHCP6OptMUD]
+    assert DHCP6OptMudUrl in pkt
+    mud_option = pkt[DHCP6OptMudUrl]
     assert mud_option.optcode == 112
```

Comment:
For the new PR: The check of the `optcode` is now probably now redunant, as the `DHCP6OptMudUrl` object gets created due to this `optcode`